Jenkinsfile

```diff
@@ -1,97 +1,67 @@
-@Library('socrata-pipeline-library') _
+@Library('socrata-pipeline-library')
 
 def dockerize = new com.socrata.Dockerize(steps, 'image-resize-service', BUILD_NUMBER)
-def marathon = new com.socrata.MarathonDeploy(steps)
+
+def lastStage
 
 pipeline {
   options {
     timeout(time: 60, unit: 'MINUTES')
     ansiColor('xterm')
   }
-
   parameters {
     string(name: 'AGENT', defaultValue: 'build-worker', description: 'Which build agent to use?')
   }
-
   agent {
     label params.AGENT
   }
-
   triggers {
     issueCommentTrigger('^retest$')
   }
-
   environment {
     SERVICE = 'image-resize-service'
-    RUNNING_IN_CI = 'true'
-    GITHUB_CREDENTIALS_ID = 'a3959698-3d22-43b9-95b1-1957f93e5a11'
-    REPOSITORY_NAME = "${env.GIT_URL.tokenize('/')[3].split('\\.')[0]}"
-    REPOSITORY_OWNER = "${env.GIT_URL.tokenize('/')[2]}"
     SERVICE_VERSION = '0.0.1'
-    ARTIFACTORY_CREDS = credentials('shared-eng-artifactory-creds')
   }
-
   stages {
     stage('Pull Request') {
       when { changeRequest() }
-      stages {
-        stage('Build') {
-          steps {
-            script {
-              sh 'sbt assembly'
-            }
-          }
-        }
-      }
-    }
-
-    stage('Initialize Job Details') {
       steps {
-        script {
-          currentBuild.description = "${env.SERVICE} : ${env.GIT_COMMIT[0..5]} : ${env.NODE_NAME}"
-        }
+        sh 'sbt assembly'
       }
     }
-
-    stage('Master') {
+    stage('Build') {
       when { branch 'main' }
       stages {
         stage('Build and push image') {
           steps {
-            sh 'sbt assembly'
+            script {
+              lastStage = env.STAGE_NAME
```

Comment:
I am inserting this lastStage variable to consolidate messages but still report the stage.  In order to do so, it needs to go in a script tag, so I've just put everything in one script tag for this stage.